language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/util/ItemResolverConstants.java

```diff
@@ -60,6 +60,7 @@
     public static final String DEFAULT = "default";
     public static final String ROLLBACK = "rollback";
 
+    public static final String ANON_FUNCTION = "anon function";
```

Comment:
Shall we use something like `function (..) {..}` as the label? Also, make sure to use function as the filter text, instead of the label itself.